src/main/webapp/search_area.js

```diff
@@ -141,11 +155,17 @@ class SearchArea {
   async getListOfOrganizations() {
     let response;
     if (this.filterParams) {
-      response = await fetch(`/list-organizations?${this.filterParams.toString()}`);
+      response = await fetch(`/list-organizations?scrs=${this.mostRecentCursor}&${this.filterParams.toString()}`);
```

Comment:
why not add this.mostRecentCursor to filterParams always?

Reply:
I did that, and then merged in this file so it fits in and works with everything that's changed about master since I first added this PR. I tested it with the most recent version of master and this PR is ready to merge!